_posts/05-05-01-PHP-and-UTF8.md

```diff
@@ -1,89 +1,65 @@
 
-[Multibyte String Extension]: http://php.net/manual/en/book.mbstring.php
-[patchwork/utf8]: https://packagist.org/packages/patchwork/utf8
+### UTF-8 a nivel de la Base de Datos
 
-### UTF-8 at the Database level
+Si su script accede a MySQL, hay grandes posibilidades de que sus cadenas sean almacenadas como cadenas No-UTF-8 aún habiendo tomado todas las precauciones anteriores.
 
-If your PHP script accesses MySQL, there's a chance your strings could be stored as non-UTF-8 strings in the database
-even if you follow all of the precautions above.
+Para asegurarse de que sus cadenas van de PHP a MySQL como UTF-8, establezca el juego de caracteres y el cotejamiento (collation) a `utf8mb4` para su base de datos y todas las tablas, use también `utf8mb4` en su cadena de conexión PDO. Vea el código de ejemplo a continuación. Esto es de _importancia crítica_.
 
-To make sure your strings go from PHP to MySQL as UTF-8, make sure your database and tables are all set to the
-`utf8mb4` character set and collation, and that you use the `utf8mb4` character set in the PDO connection string. See
-example code below. This is _critically important_.
+Tenga en cuenta de que debe usar el juego de caracteres `utf8mb4` y no `utf8` para un soporte completo UTF-8. Lea más adelante para saber porque qué.
 
-Note that you must use the `utf8mb4` character set for complete UTF-8 support, not the `utf8` character set! See
-Further Reading for why.
+### UTF-8 a nivel del Navegador
 
-### UTF-8 at the browser level
+Use la función `mb_http_output()` para asegurarse de que su script PHP genera una salida con cadenas UTF-8 hacia su navegador.
 
-Use the `mb_http_output()` function to ensure that your PHP script outputs UTF-8 strings to your browser.
-
-The browser will then need to be told by the HTTP response that this page should be considered as UTF-8. The historic approach to doing that was to include the [charset `<meta>` tag](http://htmlpurifier.org/docs/enduser-utf8.html) in your page's `<head>` tag. This approach is perfectly valid, but setting the charset in the `Content-Type` header is actually [much faster](https://developers.google.com/speed/docs/best-practices/rendering#SpecifyCharsetEarly).
+La respuesta HTTP le indicará al navegador que esa página deberá considerarse como UTF-8. La aproximación histórica para hacer esto, era incluir la [etiqueta `<meta>` charset](http://htmlpurifier.org/docs/enduser-utf8.html) en el `<head>` de su página. Esta aproximación es perfectamente válida, pero establecer el juego de caracteres en la cabecera `Content-Type` es [mucho más rápido](https://developers.google.com/speed/docs/best-practices/rendering#SpecifyCharsetEarly).
```

Comment:
Propongo para un mejor entendimiento del lector, cambiar las siguientes líneas:

- "The historic approach to doing that was to include the...": "Hace algún tiempo para realizar esto se incluía la..." 
- "This approach is perfectly valid": "Esta solución es perfectamente válida"